src/app/pages/OriginsLaunchpad/components/PreviousSalesCard/index.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { translations } from '../../../../../locales/i18n';
+import { InfoRow } from './InfoRow';
+import { CardImage } from './styled';
+import cn from 'classnames';
+
+interface IPreviousSalesCardProps {
+  saleName: string;
+  saleAllocation: string;
+  totalRaised: string;
+  participatingWallets: string;
+  date: string;
+  saleDuration: string;
+  backgroundImage: string;
+  price: string;
+  className?: string;
+}
+
+export const PreviousSalesCard: React.FC<IPreviousSalesCardProps> = ({
+  saleName,
+  saleAllocation,
+  totalRaised,
+  participatingWallets,
+  date,
+  saleDuration,
+  backgroundImage,
+  price,
+  className,
+}) => {
+  const { t } = useTranslation();
+  return (
+    <div className={cn('tw-flex tw-max-w-50', className)}>
+      <CardImage style={{ backgroundImage: `url(${backgroundImage})` }} />
+      <div className="tw-flex tw-flex-col tw-justify-center tw-ml-6 2xl:tw-ml-11">
```

Comment:
need to fix it on mobile
<img width="389" alt="Screenshot at May 31 14-30-48" src="https://user-images.githubusercontent.com/74987028/120187153-1c536680-c21d-11eb-93f4-b0cc80254fe2.png">

Reply:
Everything until 1200 px is looking good so I wouldn't fix it because we even have a message that we don't support mobile browsers yet. And almost the whole app would need to be fixed on mobile.

What do you think @t-z-u ?

Reply:
We decided to fix it and place cards in column (instead of row) on mobile devices.